docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -90,7 +90,7 @@ A collection skeleton is a directory that looks like a collection directory but
 
 An example ``galaxy.yml.j2`` file that accepts an optional dictionary variable ``dependencies`` could look like this:
 
-.. code-block:: yaml
+.. code-block:: jinja
```

Comment:
```suggestion
.. code-block:: yaml+jinja
```

Reply:
That's also fine, since `yaml+jinja` also handles YAML templated by Jinja, and not just YAML with Jinja templating in strings (what we usually have).